### DISCUSSION TITLE
add to radarr/sonarr only for specific collections

### DISCUSSION THREAD

**Discussion by lozman** [meisnate12/Plex-Meta-Manager#574] [Ideas]
Hi,

I love the feature of being able to automatically add missing items from my collections to radarr, but that can end up being hundreds if not thousands of items. What would be great is being able to specify that per collection, say for example a trending category, or what is on at the cinema right now, but not then also pull in every missing item from the dozens of actor-based collections that I have set up.

It doesn't appear that this feature exists. if it does please let me know where. Else, could it be considered for a future release?

Thanks,
Lol.

**Comment by meisnate12** [comment]
use `radarr_add` at the collection level https://github.com/meisnate12/Plex-Meta-Manager/wiki/Arr-Details

**Reply by lozman** [reply]
dang it, I am sorry for missing this. I should have known you would have thought of it since this is such comprehensive software., its really amazing. Thank you so much for creating it.

**Reply by Kyrluckechuck** [reply]
FWIW I also missed this when going through the documentation -- it might just be me struggling to read the docs but I struggled to differentiate between `radarr_add`, `radarr_add_missing`/`radarr_add_existing`, and `add_missing` 🤔 
Anywho this answer fixed it for me, thanks!

**Reply by meisnate12** [reply]
`radarr_add` is `radarr_add_missing` i renamed it but either will work (same with `add` under `radarr` being renamed to `add_missing` but the old way will still run fine)

`radarr_add_missing` - adds movies that were found but the builders of a collection that **were not in your plex** to radarr.

`radarr_add_existing` - adds movies that were found but the builders of a collection that **were already in your plex** to radarr.

`radarr_add_existing` can be combined with the `radarr_add_all` library operation to add you're whole plex to radarr.

**Reply by chancez** [reply]
Is there an example of where you actually specify these settings?
